<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="2" hostname="LAPTOP-0SJ3B8VU" failures="1" tests="1" name="ValidateTitle(failed)" time="8.69" errors="0" timestamp="2021-12-19T17:28:10 IST">
  <testcase classname="Academy.ValidateTitle" name="basePageNavigation" time="5.253">
    <failure type="java.lang.AssertionError" message="expected [A Academy to Learn Earn &amp;amp; Shine  in your QA Career] but found [An Academy to Learn Earn &amp;amp; Shine  in your QA Career]">
      <![CDATA[java.lang.AssertionError: expected [A Academy to Learn Earn & Shine  in your QA Career] but found [An Academy to Learn Earn & Shine  in your QA Career]
at Academy.ValidateTitle.basePageNavigation(ValidateTitle.java:38)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- basePageNavigation -->
  <testcase classname="Academy.ValidateTitle" name="verifyViewAllTestBoxAvailable" time="0.0">
    <ignored/>
  </testcase> <!-- verifyViewAllTestBoxAvailable -->
  <testcase classname="Academy.ValidateTitle" name="isNavigationBasDisplays" time="0.0">
    <ignored/>
  </testcase> <!-- isNavigationBasDisplays -->
</testsuite> <!-- ValidateTitle(failed) -->
